components/basic-analyzers/src/main/java/org/datacleaner/beans/CompletenessAnalyzer.java

```diff
@@ -191,7 +215,7 @@ public void fillAllConditions(Condition condition) {
         final OutputDataStreamBuilder incompleteStreamBuilder = OutputDataStreams
                 .pushDataStream(OUTPUT_STREAM_INCOMPLETE);
 
-        for (InputColumn<?> column : _valueColumns) {
+        for (InputColumn<?> column : createOutputDataStreamColumns()) {
```

Comment:
the getOutputDataStreams() method is called many times again and again, and if reusing same instance we become stale easily as configuration changes. I only wanted to use _outputDataStreamsColumns in the run(...) method because that method is called after build and initialization.